ui/app/models/pki/tidy.js

```diff
@@ -31,25 +31,45 @@ export default class PkiTidyModel extends Model {
   })
   tidyAcme;
 
+  // * auto tidy only fields
   @attr('boolean', {
     label: 'Automatic tidy enabled',
-    defaultValue: false,
```

Comment:
removed defaults so these values are retrieved from backend, this way the frontend won't accidentally override values if they change